exercises/concept/lasagna/lasagna.go

```diff
@@ -12,7 +12,7 @@ func PreparationTime(numberOfLayers int) int {
 	panic("PreparationTime not implemented")
 }
 
-// ElapsedTime calculates the total time needed to create and bake a lasagna.
+// ElapsedTime calculates the elapsed cooking time (preparation + baking) based on the amount of layers and the `actual` minutes already in the oven.
```

Comment:
```suggestion
// ElapsedTime calculates the time elapsed cooking the lasagna. This time includes the preparation time and the time the lasagna is baking in the oven.
```

I feel maybe this comment explains the function better?

Let me know what you think the current comment doesn't explain well and if this suggestion fixes it.

Reply:
Thanks! I think your suggestion makes the comment more clear :)